roles/candlepin/tasks/artemis.yml

```diff
@@ -7,42 +7,47 @@
   ansible.builtin.set_fact:
     candlepin_artemis_client_dn: "{{ openssl_response.stdout | replace('subject=', '') }}"
 
-- name: Copy Candlepin broker.xml
-  ansible.builtin.template:
-    src: broker.xml.j2
-    dest: /etc/candlepin/broker.xml
-    owner: root
-    group: tomcat
-    mode: '0640'
+- name: Create Candlepin broker.xml
+  containers.podman.podman_secret:
+    state: present
+    name: candlepin-artemis-broker-xml
+    data: "{{ lookup('ansible.builtin.template', 'broker.xml.j2') }}"
+    labels:
+      filename: broker.xml
+      app: artemis
```

Comment:
is "artemis" an app? or is it "candlepin-artemis" and "candlepin-tomcat"?

OTOH artemis runs inside tomcat too (even if not explicitly as an web-app…)

complicated.

Reply:
Yea, it's complicated... I wanted to express why it exists, what consumes it. So while it runs inside Tomcat, it's because of the internal Artemis.